StitchCoreServicesTwilio/Tests/StitchCoreServicesTwilioTests/StitchCoreServicesTwilioTests.swift

```diff
@@ -0,0 +1,16 @@
+import XCTest
```

Comment:
Can remove this file?

Reply:
Might as well keep them for now– we should have tests.